pkg/networkservice/common/connect/server_test.go

```diff
@@ -267,6 +311,61 @@ func TestConnectServer_RequestParallel(t *testing.T) {
 	}()
 }
 
+func TestConnectServer_RequestFail(t *testing.T) {
+	defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
+
+	// 1. Create connectServer
+
+	s := connect.NewServer(context.TODO(),
+		func(_ context.Context, cc grpc.ClientConnInterface) networkservice.NetworkServiceClient {
+			return injecterror.NewClient()
+		},
+		grpc.WithInsecure(),
+	)
+
+	func() {
```

Comment:
This function looks strange.

@Bolodya1997 Could we also remove this function and call `goleak.VerifyNone` in line 361?

Reply:
Yep, fixed.